### PR TITLE
Tv aanvulling db

### PR DESCRIPTION
**Pull request by astridberndsen**
Add changes and corrections on calculations and database to main

### CODE REVIEW

File: R/er_meas_rank.R [hunk 1]
```diff
@@ -166,39 +161,13 @@ er_meas_rank <- function(B_SOILTYPE_AGR, B_GWL_CLASS, A_P_SG, B_SLOPE_DEGREE, M_
...
-    dt[ec2 == 0 & eco7 == TRUE, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
+    dt[eco_app == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
     
     # lower the score  when not applicable as arable/productive/cultivated measure
-    dt[B_LU_ECO8 == TRUE & eco8 == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
-    dt[B_LU_ECO9 == TRUE & eco9 == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
-    dt[B_LU_ECO10 == TRUE & eco10 == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
+    dt[B_LU_ARABLE_ER  == TRUE & b_lu_arable_er  == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
+    dt[B_LU_PRODUCTIVE_ER == TRUE & b_lu_productive_er == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
+    dt[B_LU_CULTIVATED_ER  == TRUE & b_lu_cultivated_er == 0, c(cols) := lapply(.SD,function(x) x * 0.1), .SDcols = cols]
```
Comment (gerardhros) [inline R/er_meas_rank.R:171]: in er_meas_scores you have added also three other filters (B_LU == FALSE and b_lu_x == 1) . Is that by purpose not added here?

File: dev/ppr_tables_bbwp.R [hunk 2]
```diff
@@ -124,7 +124,7 @@ require(data.table);library(usethis)
   er_crops[,eco2:= fifelse((nc4==1|nc9==1|nc10==1|nc11==1) & (c16==1|c19==1),1,0)]
```
Comment (gerardhros) [inline dev/ppr_tables_bbwp.R:124]: these eco1 to eco7 are not needed any more, isn't?

**Review by gerardhros — APPROVED** [review]
valuable improvement. code is much easier to understand, and potentially to adapt.
two small questions left

File: dev/ppr_tables_bbwp.R (hunk 2 above)
Comment (astridberndsen) [reply dev/ppr_tables_bbwp.R:124]: That's correct. I removed the eco categories from the table.

File: R/er_meas_rank.R (hunk 1 above)
Comment (astridberndsen) [reply R/er_meas_rank.R:174]: This wasn't on purpose. I added these filters to er_meas_rank as well now.

**Review by gerardhros — APPROVED** [review]

**Pull request merged**